llama_stack/providers/remote/tool_runtime/tavily_search/tavily_search.py

```diff
@@ -69,6 +69,9 @@ async def list_runtime_tools(
 
     async def invoke_tool(self, tool_name: str, kwargs: dict[str, Any]) -> ToolInvocationResult:
         api_key = self._get_api_key()
+
+        httpx.Timeout(timeout=self.config.timeout, connect=self.config.connect_timeout)
```

Comment:
this isn't doing anything, the resulting Timeout object needs to be passed into the client